docs/cli/bootnode.md

```diff
@@ -6,7 +6,7 @@
 
 - ```v5```: Enable UDP v5 (default: false)
 
-- ```log-level```: Log level (trace|debug|info|warn|error|crit) (default: info)
+- ```verbosity```: Logging verbosity (trace|debug|info|warn|error|crit) (default: info)
```

Comment:
Verbosity was an int flag: https://github.com/maticnetwork/bor/blob/2321e64ed7f14a82776c854607148810f8389d17/cmd/evm/main.go#L88-L91 I think for consistency, we should make it an int flag in new CLI too.

Reply:
Done, thanks.